flow/or

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -ex
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+WORKSPACE=$(pwd)
+
+XSOCK=/tmp/.X11-unix
+XAUTH=/tmp/.docker.xauth
+xauth nlist :0 | sed -e 's/^..../ffff/' | xauth -f $XAUTH nmerge -
+ARGUMENTS=$@
+
+if test -t 0; then
+    DOCKER_INTERACTIVE=-ti
+fi
+
+docker run -u $(id -u ${USER}):$(id -g ${USER}) \
+ -e LIBGL_ALWAYS_SOFTWARE=1 \
+ -e "QT_X11_NO_MITSHM=1" \
+ -e XDG_RUNTIME_DIR=/tmp/xdg-run \
+ -e DISPLAY=$DISPLAY \
+ -e QT_XKB_CONFIG_ROOT=/usr/share/X11/xkb \
+ -v $XSOCK:$XSOCK \
+ -v $XAUTH:$XAUTH \
+ -e XAUTHORITY=$XAUTH \
+ -e FLOW_HOME=/OpenROAD-flow-scripts/flow/ \
+ -e MAKEFILES=/OpenROAD-flow-scripts/flow/Makefile \
+ -v $WORKSPACE:`pwd` \
+ $DOCKER_INTERACTIVE \
+ openroad/flow-centos7-builder:latest \
```

Comment:
The image name should be a parameter with a default as it is in etc/DockerHelper.sh